package.json

```diff
@@ -137,13 +137,13 @@
     "@sveltejs/adapter-vercel": "^4.0.0",
     "@sveltejs/kit": "^2.5.0",
     "@sveltejs/vite-plugin-svelte": "^3.0.2",
-    "@temporalio/activity": "^1.10.3",
-    "@temporalio/client": "^1.10.3",
-    "@temporalio/common": "^1.10.3",
-    "@temporalio/proto": "^1.10.3",
-    "@temporalio/testing": "^1.10.3",
-    "@temporalio/worker": "^1.10.3",
-    "@temporalio/workflow": "^1.10.3",
+    "@temporalio/activity": "1.10.3",
+    "@temporalio/client": "1.10.3",
+    "@temporalio/common": "1.10.3",
+    "@temporalio/proto": "1.10.3",
+    "@temporalio/testing": "1.10.3",
+    "@temporalio/worker": "1.10.3",
+    "@temporalio/workflow": "1.10.3",
```

Comment:
Locking this dependency because the newest (v1.11.1) has incorrect Nexus description types